set-GET-beta/example.env

```diff
@@ -0,0 +1,11 @@
+# The hostname to update
+#host=institution.test.instructure.com
```

Comment:
Don't need comments in the example.env do we do you think?

Reply:
it's just the standard example.env file so I'll leave it.